client/scss/layouts/_login.scss

```diff
@@ -122,4 +122,10 @@
     margin: theme('spacing.8') auto 0;
     width: theme('spacing.8');
   }
+
+  .login-logo-img{
+    @media (forced-colors: active) and (prefers-color-scheme: dark){
```

Comment:
One more tweak - search through the code for other times we set forced-colors and you'll see that we use a variable for the `active` - it would be good to adopt this variable here also.